crates/astria-conductor/src/data_availability/block_verifier.rs

```diff
@@ -45,9 +45,9 @@ impl BlockVerifier {
     }
 
     pub(super) async fn validate_sequencer_blob(
-        &self,
-        blob: &CelestiaSequencerBlob,
-    ) -> eyre::Result<()> {
+        self,
+        blob: CelestiaSequencerBlob,
+    ) -> eyre::Result<CelestiaSequencerBlob> {
```

Comment:
is ownership of blob necessary here?

Reply:
It made for a nicer flow while rewriting this (I had some `async move {}` that's no longer there), but it's technically not necessary anymore. It gets reworked in a follow-up PR here though so I am going to leave this -> https://github.com/astriaorg/astria/pull/698